wiki/Tournaments/CIC/2018/en.md

```diff
@@ -0,0 +1,373 @@
+---
+tags:
+- CIC 2018
+- CIC2018
+---
+# Catch the Beat Indonesia Cup 2018
+
+![CIC 2018 Logo](img/logo.jpg)
+
+The **Catch the Beat Indonesia Cup 2018** (***CIC 2018***) is Indonesian one-on-one osu!catch tournament hosted by ![][flag_ID] [\[490\]Kazuyo](/users/1830361). It is the 6th installment of the Catch the Beat Indonesia Cup 2018.
```

Comment:
`is Indonesian` -> `is an Indonesian`

`6th installment of the Catch the Beat Indonesia Cup 2018` -> `6th installment of the Catch the Beat Indonesia Cup`